### DISCUSSION TITLE
How can I use pywal colors?

### DISCUSSION THREAD

**Discussion by jyyhuang** [wezterm/wezterm#5803] [Q&A]
### What Operating System(s) are you running on?

Linux Wayland

### Which Wayland compositor or X11 Window manager(s) are you using?

Hyprland

### WezTerm version

20240701-070926-69686f45

### Ask your question!

As the title states, I was wondering how I could use pywal colors to customize the tab bar.

**Comment by wez** [comment]
Maybe check out what the OP in https://github.com/wez/wezterm/issues/1036 is doing
